cursorless-talon/src/actions/call.py

```diff
@@ -1,13 +1,23 @@
-from talon import actions
+from typing import Optional
+
+from talon import Module, actions
 
 from ..targets.target_types import CursorlessTarget, ImplicitTarget
 
+mod = Module()
+mod.list("cursorless_call_action", desc="Cursorless call action")
 
-def cursorless_call_action(target: CursorlessTarget):
-    actions.user.private_cursorless_command_and_wait(
-        {
-            "name": "callAsFunction",
-            "callee": target,
-            "argument": ImplicitTarget(),
-        }
-    )
+@mod.action_class
+class Actions:
+    def private_cursorless_call(
+        callee: CursorlessTarget,
+        argument: Optional[CursorlessTarget] = ImplicitTarget(),
```

Comment:
Does this actually accept `None`?
```suggestion
        argument: CursorlessTarget = ImplicitTarget(),
```

Reply:
We either call this function with one or two arguments. There is no reason to pass none.

Reply:
If there's no reason to pass `None` to this, then this shouldn't be type hinted to accept `None` :)

Reply:
I don't follow?

Reply:
Whether something is optional (i.e. can be `None`) is orthogonal to whether an argument has a default value. https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
` typing.Optional` `Note that this is not the same concept as an optional argument`. Of course it isn't! :D
Thanks!